Userland/Libraries/LibWeb/Loader/FrameLoader.cpp

```diff
@@ -157,6 +157,11 @@ bool FrameLoader::load(const LoadRequest& request, Type type)
         return false;
     }
 
+    if (!m_frame.is_frame_nesting_allowed(request.url().to_string())) {
```

Comment:
`is_frame_nesting_allowed()` already takes a `URL`, so I don't think we need to `to_string()` here. That will only cause an unnecessary URL->String->URL round-trip.